src/composables/state.ts

```diff
@@ -0,0 +1,497 @@
+import VueCompositionAPI, { computed, ref } from '@vue/composition-api'
```

Comment:
We could move this into seperate files if needed. The other two stores are fairly small.